controllers/csi/driver/server.go

```diff
@@ -256,14 +265,27 @@ func (svr *CSIDriverServer) NodeUnpublishVolume(ctx context.Context, req *csi.No
 	}
 
 	if err := BindUnmount(
-		Mount{Target: filepath.Join(targetPath, "agent", "conf")},
-		Mount{Target: filepath.Join(targetPath, "log")},
-		Mount{Target: filepath.Join(targetPath, "datastorage")},
+		Mount{Target: filepath.Join(targetPath, dtcsi.AgentConfDir)},
+		Mount{Target: filepath.Join(targetPath, dtcsi.LogDir)},
+		Mount{Target: filepath.Join(targetPath, dtcsi.DatastorageDir)},
 		Mount{Target: targetPath},
 	); err != nil {
 		return nil, status.Error(codes.Internal, fmt.Sprintf("failed to unmount volume: %s", err.Error()))
 	}
 
+	linkFile := filepath.Join(dtcsi.GarbageCollectionPath, volumeID)
```

Comment:
Path points to` /gc/<volumeId`, like previously, are you sure that is the correct path?

Reply:
Changed